cfg/config.yaml

```diff
@@ -78,15 +78,26 @@ node:
     - kubernetes
 
   kubernetes:
-    defaultconf: /etc/kubernetes/config    
+    confs:
+      - "/var/lib/kubelet/kubeconfig"
+      - "/etc/kubernetes/config"
+    defaultconf: "/etc/kubernetes/config"
 
   kubelet:
     cafile:
       - "/etc/kubernetes/pki/ca.crt"
+      - "/etc/kubernetes/certs/ca.crt"
+    svc:
+      - "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
+      - "/etc/systemd/system/kubelet.service"
     bins:
       - "hyperkube kubelet"
       - "kubelet"
+    kubeconfig:
+      - "/var/lib/kubelet/kubeconfig"
```

Comment:
Since the file locations are looked at in order, I would prefer the newly-added location to be added at the end of the list (i.e. add `/var/lib/kubelet/kubeconfig` after `/etc/kubernetes/kubelet.conf`) as it reduces the risk of this fix breaking another platform.